packages/scandipwa/src/component/MyAccountMyWishlist/MyAccountMyWishlist.container.tsx

```diff
@@ -85,9 +93,36 @@ S extends MyAccountMyWishlistContainerState = MyAccountMyWishlistContainerState,
             isLoading: false,
             loadingItemsMap: {} as Record<string, boolean>,
             isQtyUpdateInProgress: false,
+            productsPerPage: 10,
         } as S;
     }
 
+    componentDidMount(): void {
+        const { updateWishlistProducts, location } = this.props;
+        const { productsPerPage } = this.state;
+        const page = getPageFromUrl(location);
```

Comment:
At one point we have refactored to use location from history util, instead of react-router. So can safely remove HOC and just import history from util. This is done for consistency.